url_screenshot/plugin.spec.yaml

```
@@ -0,0 +1,50 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: url_screenshot
+title: Screenshot a URL
+description: Screenshot a URL from the Orchestrator
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/url_screenshot
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- utilities
+- url
+hub_tags:
+  use_cases: [data_enrichment]
+  keywords: [utilities, analysis, url]
+  features: []
+actions:
+  screenshot:
+    title: Screenshot URL
+    description: Takes a screenshot of a URL from the Orchestrator
+    input:
+      url:
+        title: URL
+        type: string
+        description: URL to screenshot
+        required: true
+        order: 1
+      delay:
+        title: Delay
+        type: integer
+        description: Wait x seconds after page load before taking screenshot
```

```suggestion
        description: Delay (in seconds) after page has loaded before screenshot
```
Thoughts? Just trying to make the description more succinct.